### DISCUSSION TITLE
Does Cake.MarkdownToPdf work with Cake.Frosting?

### DISCUSSION THREAD

**Discussion by mgnslndh** [cake-build/cake#3489] [Extension Q&A]
Cake.MarkdownToPdf works fine with scripted Cake but I can't get it to work using Cake.Frosting. In Frosting I use a `<PackageReference/>` but that won't copy the themes bundled with Cake.MarkdownToPdf. 

I guess I can set the paths manually but I'm wondering if I'm doing anything wrong or if there might be a bug causing themes to not work in Frosting.

I'm using Cake.MarkdownToPdf 2.5.2

**Comment by mgnslndh** [comment]
I guess the NuGet package needs to be fixed for better integration with Cake.Frosting. Right now the `wkhtmltopdf.exe` dependency is located in the lib folder which cause VS to load it as an assembly:

```
 warning MSB3246: Resolved file has a bad image, no metadata, or is otherwise inaccessible. System.BadImageFormatException: Could not load file or assembly 'wkhtmltopdf.exe' or one of its dependencies. The module was expected to contain an assembly manifest.
 warning MSB3246: File name: 'wkhtmltopdf.exe' ---> System.BadImageFormatException: Could not load file or assembly 'wkhtmltopdf.exe' or one of its dependencies. The module was expected to contain an assembly manifest.
 warning MSB3246: File name: 'wkhtmltopdf.exe'
 warning MSB3246: 
 warning MSB3246: 
 warning MSB3246:    at System.Reflection.AssemblyName.nGetFileInformation(String s)
 warning MSB3246:    at System.Reflection.AssemblyName.GetAssemblyName(String assemblyFile)
 warning MSB3246:    at Microsoft.Build.Shared.AssemblyNameExtension.GetAssemblyNameEx(String path)
 warning MSB3246:    at Microsoft.Build.Tasks.SystemState.GetAssemblyName(String path)
 warning MSB3246:    at Microsoft.Build.Tasks.ReferenceTable.SetPrimaryAssemblyReferenceItem(ITaskItem referenceAssemblyName)
 warning MSB3246: 
 warning MSB3246:
```

**Reply by nils-a** [reply]
I'm seeing the same thing, and I agree with your assessment that this is due to the `exe` being placed inside the `lib` folder of the package. Are you in a position to open an issue over at https://github.com/twenzel/Cake.MarkdownToPdf/issues ?

**Reply by mgnslndh** [reply]
Yes, I can do that.

**Reply by augustoproiete** [reply]
Pinging @twenzel for awareness

**Reply by mgnslndh** [reply]
I submitted two separate issues:

- https://github.com/twenzel/Cake.MarkdownToPdf/issues/45
- https://github.com/twenzel/Cake.MarkdownToPdf/issues/46
